### DISCUSSION TITLE
Updating from v3 to v7

### DISCUSSION THREAD

**Discussion by UVLabs** [actions/github-script#463] [Q&A]
I've only just now seen the Annotation that github action adds beneath runs and realized that the action needs to be updated because of the deprecating of old node versions by Github action.

I use this action to create a tag:

```
       name: Create Tag
       uses: actions/github-script@v3
       env:
          TAG: ${{ steps.set-tag.outputs.tag_name }}
       with:
          github-token: ${{ github.token }}
          script: |
            github.git.createRef({
              owner: context.repo.owner,
              repo: context.repo.repo,
              ref: "refs/tags/${{ steps.set-tag.outputs.tag_name }}",
              sha: context.sha
            })
```

I'm wondering if there's any breaking change that would cause this same code to not work? I have this integrated into an action that does a release and pushes to an external server where users download the build; I'm worried about updating to v7 and then having the wrong code be sent out to users if no exit error happens but yet something went wrong...

Looking for some insight as this would be a big jump from v3 - v7

**Comment by joshmgross** [comment]
Octokit changed in v5, you'll want to use `github.rest.git.createRef`
See https://github.com/actions/github-script#v5 for more details.
